plugins/copy-to-clipboard/prism-copy-to-clipboard.js

```diff
@@ -50,7 +52,7 @@
 		function registerClipboard() {
 			var clip = new ClipboardJS(linkCopy, {
 				'text': function () {
-					return env.code;
+					return element.textContent;
```

Comment:
Can we move this out of this PR and come back to it later in #1974.

Reply:
Sure but why would this be part of #1974?

Reply:
Because #1974 changes File Highlight and I'd like to look at fixing the issue by updating `env.code` when the file gets highlighted.